### ISSUE TITLE
Unable to boot after updating packages.

### ISSUE THREAD

**Issue by ReptoxX** [pop-os/pop#563]
**Distribution (run `cat /etc/os-release`):**
NAME="Pop!_OS"
VERSION="19.04"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Pop!_OS 19.04"
VERSION_ID="19.04"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=disco
UBUNTU_CODENAME=disco

 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
linux-generic:
  Installed: 5.0.0.15.16pop0
  Candidate: 5.0.0.15.16pop0
  Version table:
 *** 5.0.0.15.16pop0 1001
       1001 http://ppa.launchpad.net/system76/pop/ubuntu disco/main amd64 Packages
        100 /var/lib/dpkg/status
     5.0.0.15.16 500
        500 http://us.archive.ubuntu.com/ubuntu disco-security/main amd64 Packages
        500 http://us.archive.ubuntu.com/ubuntu disco-updates/main amd64 Packages
     5.0.0.13.14 500
        500 http://us.archive.ubuntu.com/ubuntu disco/main amd64 Packages

**Issue/Bug Description:**
Since updating my kernel version yesterday, i cant boot into the os anymore.

**Steps to reproduce (if you know):**
sudo apt update
sudo apt upgrade -y

**Expected behavior:**
To update all the packages and beeing able to boot properly

**Other Notes:**
I am not sure if its related to pop os, but since i only have this issue with this os i am posting it here.

**Comment by ReptoxX** [comment]
Due to the fact that i "Refresh Installed" the system and hold the kernel update back and i could still not boot from it again, i suggest it's not the kernel version.
But because i am not this into linux, i am not sure which package it is..

I've uploaded the list of upgrades i did, into this pastebin for further examination.
https://pastebin.com/p8GWN5PF

**Comment by point-source** [comment]
When you say you cannot boot, where does the boot stop? Do you get a bios splash and then a black screen? If your drive is encrypted, do you get prompted to enter the password at least? Is there any kind of visible error or artifact on the screen?

**Comment by ReptoxX** [comment]
There are no errors displayed onscreen and no artifacts.
I've tried to boot with selecting the boot device in the menu and the menu didnt even disappear.
Unfortunatly i don't use encrypted drives, so no password required

**Comment by mmstick** [comment]
What's your hardware specs?

**Comment by ReptoxX** [comment]
CPU: Intel Core i7 6800k
Memory: 16GB DDR4
GPU: Nvidia Geforce GTX 1080 TI
HDD: Samsung NVMe

**Comment by mmstick** [comment]
Unsure if this would help you, but you could try running `kernelstub -a 'nvidia-drm.modeset=0' to disable kernel modesetting. Modesetting is required to fix screen tearing and enable Wayland support, but a few people have reported issues with it being enabled.

**Comment by ReptoxX** [comment]
I can give it a shot, but everytime i update my system and have to restore it, i have to reinstall all my programs and stuff. So it's it's quite exhausting.
But still i am trying it tomorrow, if someone else has an idea that i can try within the same attempt, please let me know.

**Comment by point-source** [comment]
Just for reference, I encountered a very similar issue to this after I took the latest updates as well. I am using a Lenovo X1 Extreme with PopOS 19.04 that had been upgraded from 18.10. 

After the most recent updates, the system appeared not to boot. Despite the black screen though, it would still accept my decryption password for my encrypted drive and if I input it correctly, the system would eventually boot into gnome. 

Powering off took a very long time (~5mins) while it was hung on a black screen with a cursor that was occasionally frozen. Additionally, attempting to switch graphics would claim failure with an error message but would sometimes still switch after a reboot. This behavior was inconsistent.

Two days ago I did a fresh install of 19.04 on a secondary drive. I spent yesterday getting everything set up again the way I had it and am happy to report that it appears to be working fine now on the latest packages.
